CMakeLists.txt

```diff
@@ -169,8 +169,13 @@ SET(CPACK_DEBIAN_PACKAGE_SECTION "Games")
 
 # RPM specific settings
 SET(CPACK_RPM_PACKAGE_GROUP "Amusements/Games")
-SET(CPACK_RPM_PACKAGE_AUTOREQ ON)
+
+SET(CPACK_RPM_PACKAGE_REQUIRES "cairo-devel, librsvg2-devel, glibmm24-devel, libxml++-devel, boost-devel, SDL2-devel, libepoxy-devel, ffmpeg-devel, portaudio-devel, opencv-devel, portmidi-devel, libjpeg-turbo-devel, pango-devel, libpng-devel, glm-devel, openblas-devel, fftw-devel, cpprest-devel")
 SET(CPACK_RPM_PACKAGE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/licence.txt")
+message("${LC_MESSAGES_DIRS}")
+message("HOOOOOI")
```

Comment:
Dutch for "hello?"

Reply:
Yup had to sync between laptops this is gonna get squashed/deleted :joy: